init.js

```diff
@@ -520,3 +520,17 @@ commands['Define word'] = {
 // this essentially removes a lot of the distractions from your view in VS Code.
 // (As always, toggle this with Cmd+Shift+P ---or Ctrl+X+Z)
 E.executeCommand('workbench.action.toggleZenMode')
+
+function quoteSelection(quote = "\"", sep = " ") {
```

Comment:
Some discussion on the merits of including these would be appreciated.
- Even a single-line comment justifying why you want these would be nice.
- Also, you can replace your 5-line function with a single line using `E.replaceSelection`.
   + A nearly identical implementation to your function already exists here, https://github.com/alhassy/easy-extensibility/blob/main/init.js#L42, so maybe you could consider altering that 🤷‍♂️

Reply:
Thanks, this is just making more customization to the quote operation.
For example, if one just want to turn the command line "-i a.file -o b.out -c xxx -d yyy" into some json array ["-i", "a.file", "-o", "b.out", "-c", "xxx", "-d", "yyy"] which existed in the tasks.json